src/lib/data/helpers.ts

```
@@ -0,0 +1,200 @@
+      // Start of current week (considering Sunday as first day)
+      const day = now.getDay(); // 0 for Sunday, 1 for Monday, etc.
+      startDate = new UTCDate(now);
+      startDate.setDate(now.getDate() - day);
+      break;
+    case "monthly":
+      // Start of current month
+      startDate = new UTCDate(now.getFullYear(), now.getMonth(), 1);
+      break;
+    case "quarterly":
+      // Start of current quarter
+      const month = now.getMonth();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)